Assets/MRTK/Providers/Oculus/XRSDK/Controllers/OculusXRSDKTouchController.cs

```diff
@@ -88,5 +89,27 @@ protected override void UpdateButtonData(MixedRealityInteractionMapping interact
                 }
             }
         }
+
+        /// <summary>
+        /// Determines whether or not this controller uses MRTK for controller visualization or not
+        /// </summary>
+        public bool UseMRTKControllerVisualization = false;
```

Comment:
Can this be made `internal`? Also, could this be converted to a property as we've typically only had private fields

```suggestion
        internal bool UseMRTKControllerVisualization { get; set; } = false;
```